sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -433,6 +433,18 @@ def decode(self, encoded):
     return encoded
 
 
+class StrUtf8CoderImpl(StreamCoderImpl):
+  """For internal use only; no backwards-compatibility guarantees."""
+  def encode_to_stream(self, value, out, nested):
+    byte_value = value.encode('utf-8')
+    out.write_var_int64(len(byte_value))
+    out.write(byte_value)
+
+  def decode_from_stream(self, in_stream, nested):
+    byte_length = in_stream.read_var_int64()
```

Comment:
Similarly. Otherwise read to the end of the stream. This can be done with `stream.read_all(nested)`.